ztp/gitops-subscriptions/argocd/deployment/openshift-gitops-operator.yaml

```diff
@@ -9,9 +9,8 @@ metadata:
   name: openshift-gitops-operator
   namespace: openshift-operators
 spec:
-  channel: stable
+  channel: gitops-1.8
```

Comment:
shall we consider using "latest" channel instead of versioned channel?

Reply:
Well, the idea to stay with latest-1 version and update the channel with every release.